Docs/Edit-UserSettingsjson.md

```diff
@@ -29,46 +31,52 @@ When Hurl is lauched for the first time, it automatically detects the installed
 }
 ```
 
-## App Settings
+## App settings
+
+The following snippet shows the default options:
 
-The following snippet does this shows the default options
 ```json
 "AppSettings": {
-  "LaunchUnderMouse": false,
-  "RuleMatching": false,
-  "NoWhiteBorder": false,
-  "BackgroundType": "mica",
-  "WindowSize": [420,210]
+    "LaunchUnderMouse": false,
+    "MinimizeOnFocusLoss": true,
+    "NoWhiteBorder": false,
+    "BackgroundType": "mica",
+    "RuleMatching": false,
+    "WindowSize": [460,210]
 }
 ```
 
-### Available Options
-- `LaunchUnderMouse` defaulted to false, can be used to launch the hurl window under the mouse when enabled
-- `NoWhiteBorder` set true or false to enable or disable the white border around the window.
-- `BackgroundType` supports `mica`, `acrylic`, `none`
-  - win11 22H2 or above supports all options
-  - win11 build 22000 only mica
-  - win10 is defaulted to `none` irrespective of option
-- `WindowSize` is to set the size of Hurl BrowserSelect window. You dont need to set this, Just resize the window accordingly, It get filled automatically.
-- `RuleMatching` defaulted to false, on enabling, it supports the features from [Rulesets](https://github.com/U-C-S/Hurl/wiki/Rulesets)
-
-## Browsers Field
-
-- `Name` - you know (Req.)
-- `ExePath` - The path of browser main exe file (Req.)
-- `CustomIconPath` - Use absolute path of the image. Also, Supports URLs (OPTIONAL)
-- `LaunchArgs` - Add the default exe launch arguments here. use param `%URL%` injecting the Url at runtime here (OPTIONAL)
-- `Hidden` - set it to `true` to hide the current icon in the selection screen (OPTIONAL)
-- `AlternateLaunches` - This is an array .... see below (OPTIONAL)
+### Available options
+
+- `LaunchUnderMouse` default is **false**, can be used to launch the Hurl window under the mouse when enabled
+- `MinimizeOnFocusLoss` default is **true**
+- `NoWhiteBorder` set **true** or **false** to enable or disable the white border around the window
```

Comment:
This should be looked at: do you **show** or **hide** the border? And is it even there in the latest UI?

Reply:
> show or hide the border?

Yes, the one around the select browser window

Reply:
I mean: true = show or true = hide?

Reply:
Oh I see, I worded it confusing. 

true=hide